cvat/apps/engine/models.py

```diff
@@ -205,6 +205,12 @@ class Data(models.Model):
     sorting_method = models.CharField(max_length=15, choices=SortingMethod.choices(), default=SortingMethod.LEXICOGRAPHICAL)
     deleted_frames = IntArrayField(store_sorted=True, unique_values=True)
 
+    # Avoid storing whole mapping here, its redundant
+    custom_segments = models.BooleanField(default=False)
```

Comment:
Can't this be determined by checking if `segment_size` is 0 on the task? Maybe we don't need this new field?

Reply:
I also thought that this variable can be excluded. Basically, we'd need to check the segment size and the number of jobs != 1. However, it turned out that in the case of 1 job in the task, the cases are hard to distinguish. I've decided then to add a clear indicator instead.

Reply:
Is it possible to have non-zero `segment_size` when there's no custom mapping? I thought `segment_size` defaulted to the number of frames.

Reply:
> Is it possible to have non-zero segment_size when there's no custom mapping?

Yes, it represents limitations on the size of a single job. The comment says about this field:

> Zero means that there are no limits (default)